Xamarin.Forms.Core/Element.cs

```diff
@@ -158,7 +159,8 @@ internal IPlatform Platform
 		}
 
 		// you're not my real dad
-		internal Element RealParent { get; private set; }
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public Element RealParent { get; private set; }
```

Comment:
The controller interface declares RealParent so it is on the controller interface.